matrimony_admin/views.py

```
@@ -1,16 +1,22 @@
 import json
 import os
 from django.http import HttpRequest, HttpResponse
-from django.shortcuts import render, redirect
-from django.views.generic import TemplateView, DetailView, ListView
+from django.shortcuts import render, redirect, get_object_or_404, get_object_or_404
+from django.views.generic import TemplateView, DetailView, ListView,DeleteView,UpdateView
 from django.contrib import messages
 from django.contrib.auth import authenticate, login, logout
-from django.views.generic import FormView
+from django.views.generic import FormView, ListView
```

Ruff

Redefinition of unused `ListView` from line 5

Remove definition: `ListView`

(F811)